packages/corelight/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.5.1"
```

```suggestion
- version: "0.6.0"
```
If this is enhancement, lets go with incrementing major as suggested. 
If its bugfix, change `type: bugfix` and retain version.

---

sure 👍